packages/svelte/package.json

```diff
@@ -74,11 +74,12 @@
     "format": "prettier . --cache --plugin-search-dir=. --write",
     "check": "prettier . --cache --plugin-search-dir=. --check",
     "test": "vitest run && echo \"manually check that there are no type errors in test/types by opening the files in there\"",
-    "build": "rollup -c && npm run tsd",
-    "prepare": "npm run build",
+    "build": "rollup -c && pnpm tsd",
+    "prepare": "pnpm build",
+    "generate:version": "pnpm build",
```

Comment:
ooooh, would this also mean we'd be commiting the generated types?

Reply:
separated that out so that we're only generating the version now